src/include/duckdb/common/types/row/tuple_data_collection.hpp

```diff
@@ -229,7 +229,7 @@ class TupleDataCollection {
 	//! The number of entries stored in the TupleDataCollection
 	idx_t count;
 	//! The data segments of the TupleDataCollection
-	vector<TupleDataSegment> segments;
+	vector<TupleDataSegment, false> segments;
```

Comment:
Can we alias this to `unsafe_vector` instead to be more explicit? Something like:

```sql

template <typename T>
using unsafe_vector = vector<T, false>;
```